openshift/templates/eagle-api.dc.json

```diff
@@ -201,28 +201,6 @@
                     "name": "API_HOSTNAME",
                     "value": "${APPLICATION_DOMAIN}"
                   },
-                  {
```

Comment:
These `MINIO_*` fields need to stay because they are still used for the S3 connection.

Reply:
See 5933c01